src/components/roadmap/RoadmapPanel.svelte

```diff
@@ -36,7 +42,7 @@
         {/if}
     </div>
     <div class="content-section">
-        {#each content as item}
+        {#each content as item, i (`${key}-content-${i}`)}
```

Comment:
Can we add a content id so there is a uniquely identifying field for each element in the content array?
We can then change the loop to the following:
```suggestion
        {#each content as item (content.id)}
```